README.md

```diff
@@ -1,192 +1,66 @@
-# SuperEditText
->  Click to see [English Document](https://github.com/Carson-Ho/SuperEditText/blob/master/README-en.md)
-- 作者：Carson_Ho
-- 概述
+# Super_EditText
 
-![示意图](http://upload-images.jianshu.io/upload_images/944365-ff40bf46370cd596.png?imageMogr2/auto-orient/strip%7CimageView2/2/w/1240)
+A HMOS library which provides a simple & useful DIY View for EditText
```

Comment:
Can modify the description to - "A HMOS library which provides SuperEditText with one-click delete feature & custom styles"

Reply:
Done